source/docs/yearly-overview/known-issues.rst

```diff
@@ -15,6 +15,26 @@ Onboard I2C Causing System Lockups
 
 **Workaround:** The only surefire mitigation is to use the MXP I2C port instead. Acessing the device less frequently and/or using a different roboRIO may significantly reduce the likelihood/frequency of lockups, it will be up to each team to assess their tolerance of the risk of lockup. This lockup can not be definitively identified on the field and a field fault will not be called for a match where this behavior is believed to occur. This lockup is a CPU/kernel hang, the roboRIO will completely stop responding and will not be accessible via the DS, webpage or SSH. If you can access your roboRIO via any of these methods, you are experiencing a different issue.
 
+Analog Devices Gyros don't work properly
```

Comment:
I would either capitalize all of it or none seems arbitrary currently.  This applies to each of the headings